lib/ontologies_linked_data/config/config.rb

```diff
@@ -103,6 +103,88 @@ def config(&block)
     # number of threads to use when indexing a single ontology for search
     @settings.indexing_num_threads          ||= 1
 
+
+
+    # Global connector configuration
+    @settings.connectors ||= {
+      available_sources: {
+        'ANR' => Connectors::AnrConnector,
+        'CORDIS' => Connectors::CordisConnector
+      }
+    }
+    @settings.project_sources ||= ["ANR", "CORDIS"]
```

Comment:
this does not to need to be configurable, you can infer it from the list of connectors